phoenix-core/src/main/java/org/apache/phoenix/iterate/ExplainTable.java

```diff
@@ -147,7 +148,20 @@ protected void explain(String prefix,
             scanTypeDetails = explainSkipScan();
         }
         buf.append(scanTypeDetails);
-        buf.append("OVER ").append(tableRef.getTable().getPhysicalName().getString());
+
+        String tableName = tableRef.getTable().getPhysicalName().getString();
+        if (ScanUtil.isLocalIndex(scan)) {
```

Comment:
Rather than `ScanUtil#isLocalIndex`, we should check PTable index type:
```
    @Override
    public IndexType getIndexType() {
        return indexType;
    }
```